### PR TITLE
revamp of multi-app and git submodules doc pages

### PR DESCRIPTION
**Pull request by flovntp**
## Why

Closes [#{ISSUE_NUMBER}](https://github.com/platformsh/platformsh-docs/issues/3235)

### CODE REVIEW

File: docs/src/create-apps/multi-app/routes.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+all of your apps are served by a single [router for the project](/define-routes/_index.md).
+Each of your apps must have a `name` that's unique within the project.
+To define specific routes for one of your apps, use this `name`.
+
+There are various ways you can define routes for multiple app projects such as the following example:
+
+![Diagram of a project containing multiple apps](/images/config-diagrams/multiple-app.png "0.5")
+
+In this project, you have a CMS app, two frontend apps (one using Symfony and another using Gatsby)
+and a Mercure Rocks server app, defined as follows:
```
Comment (AnouckColson) [inline docs/src/create-apps/multi-app/routes.md:21]:
```suggestion
In this project, you have a CMS app, two frontend apps (one using Symfony and another using Gatsby),
and a Mercure Rocks server app, defined as follows:
```

File: docs/src/create-apps/multi-app/routes.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  source:
+    root: mercure/.config
+  ...
+```
+
+{{< note >}}
+
+You don't need to define a route for each app in the repository.
+If an app isn't specified, then it isn't accessible to the web.
+One good example of defining an app with no route is when you use Git submodules and want to enable
```
Comment (AnouckColson) [inline docs/src/create-apps/multi-app/routes.md:50]: @flovntp Unfinished sentence here.

File: docs/src/development/submodules.md [hunk 3]
```diff
@@ -87,46 +263,45 @@ If your server needs access to multiple repositories, follow these steps:
...
 
 {{< /note >}}
 
 ## Removing submodules
 
 These steps aren't specific to Platform.sh, but kept as a reference for Git so that submodules are effectively removed before entering the build process.
 
-1. Delete information for the submodule you'd like to remove from `.gitmodules`.
-2. Stage changes to `.gitmodules`: 
+1. Delete information for the submodule you'd like to remove from `.gitmodules` and `.git/config`.
```
Comment (AnouckColson) [inline docs/src/development/submodules.md:274]:
```suggestion
1. In your `.gitmodules` and `.git/config` files, delete the information related to the submodule you want to remove.
```

**Review by AnouckColson — CHANGES_REQUESTED** [review]

**Comment by AnouckColson** [comment]
@flovntp Doc review done, left a few last minute suggestions/comments. Let me know if you see anything amiss!

File: docs/src/create-apps/multi-app/routes.md (hunk 2 above)
Comment (flovntp) [reply docs/src/create-apps/multi-app/routes.md]: finished and pushed

File: docs/src/create-apps/multi-app/routes.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  source:
+    root: mercure/.config
+  ...
+```
+
+{{< note >}}
+
+You don't need to define a route for each app in the repository.
+If an app isn't specified, then it isn't accessible to the web.
+One good example of defining an app with no route is when you [use Git submodules](/create-apps/multi-app/project-structure.html#split-your-code-source-into-multiple-git-submodule-repositories) and want to [use Source operation to update your submodules](/development/submodules.html#update-submodules).
```
Comment (AnouckColson) [inline docs/src/create-apps/multi-app/routes.md:50]:
```suggestion
One good example of defining an app with no route is when you [use Git submodules](/create-apps/multi-app/project-structure.html#split-your-code-source-into-multiple-git-submodule-repositories) and want to [use a source operation to update your submodules](/development/submodules.html#update-submodules).
```

**Review by AnouckColson — CHANGES_REQUESTED** [review]

**Review by thomasdiluccio — DISMISSED** [review]

**Pull request merged**